### PR TITLE
Adds the interactive hamburger feature on navbar

### PR DESCRIPTION
**Pull request by mkaya13**
Adds the interactive hamburger feature to navbar

### CODE REVIEW

**Comment by mkaya13** [comment]
I changed the interactive navbar options a little bit in purpose since I did not like the one in the figma.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Mert, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are passing

## Required Changes ♻️
- I read your comment regarding the styling, however, it is a requirement to follow the figma design as close as possible, the reason for this is to learn how to build a webpage after an exact example in the way it might be requested in your future job.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -38,10 +39,6 @@ Through Github
 - Linter test in local:
   `npx stylelint "**/*.{css,scss}"`
 
-### Deployment
-
-- Not deployed yet
-
 ## Authors
```
Comment (rloterh) [inline README.md:42]: - [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

File: README.md [hunk 2]
```diff
@@ -38,10 +39,6 @@ Through Github
...
 
 👤 **Author1**
```
Comment (rloterh) [inline README.md:44]: - [ ] OPTIONAL
Generally, the subheading text Author1 should be replaced with your full name in your ReadMe.

- [ ] OPTIONAL
MIT license link on _line 64_ of your ReadMe is broken, kindly endeavor to fix it.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mert,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾Congratulations!  🎉💯🌟 

## Highlights 👍
- Nice job for using cursor pointer for the mobile menu
- Mobile menu is working as intended

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**